sdk/storage/azure-storage-blob/setup.py

```diff
@@ -76,14 +76,14 @@
     ]),
     python_requires=">=3.7",
     install_requires=[
-        "azure-core<2.0.0,>=1.26.0",
+        "azure-core<2.0.0,>=1.28.0",
         "cryptography>=2.1.4",
-        "typing-extensions>=4.0.1",
+        "typing-extensions>=4.3.0",
```

Comment:
Need to bump this because previous commit was failing for this reason: 
``` 
The conflict is caused by:
    The user requested typing-extensions==4.0.1
    azure-core 1.28.0 depends on typing-extensions>=4.3.0
```